packages/web/stores/derived-data/concentrated-liquidity/historical-and-liquidity-data.ts

```diff
@@ -105,7 +108,7 @@ export class ObservableHistoricalAndLiquidityData {
     if (!this.pool || this.pool.type !== "concentrated") return new Dec(0);
     return (
       this.pool.concentratedLiquidityPoolInfo?.multiplicationQuoteOverBase ??
-      new Dec(0)
+      new Dec(1)
```

Comment:
Why did this change?

Reply:
When the pool is loading, the spot price is 0, causing the multiplicationQuoteOverIn to be 0, causing divide by 0 exception in the depthChart getter